timemachine/potentials/generic.py

```diff
@@ -22,6 +23,7 @@
 BondedGpuPotential = TypeVar("BondedGpuPotential", bound=gpu.BondedWrapper)
 
 
+@typing_extensions.runtime
```

Comment:
This seems to be a [deprecated alias]((https://github.com/python/typing_extensions/blob/2979419f7bb10b554102e972351f8d03c8ffd1fa/src/typing_extensions.py#L635)) for `runtime_checkable`. Should we use the latter instead?

Reply:
willfix